### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by ygoyal18**
Follows - https://github.com/armPelionEdge/meta-pelion-edge/pull/148

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,90 +1,5 @@
...
-
-To flash on Linux, use dd.  You can use `lsblk` to find out the name of your SD card block device.
-
-        $ gunzip -c console-image-raspberrypi3.rootfs.wic.gz | sudo dd bs=4M of=/dev/mmcblkX conv=sync
-
-
-Troubleshooting
----------------
-1. See the build-pelion-edge [README](https://github.com/armpelionedge/build-pelion-edge/blob/master/README.md) for solutions to some common build errors.
+For more information, please see the [Pelion Edge documentation](https://www.pelion.com/docs/device-management-edge/latest/introduction/index.html).
```
Comment (costanic) [inline README.md:5]: This removes a great deal of helpful information which isn't replaced by this link.  Can you add a link to the documentation where these steps are covered?  As a user, if I'm already looking at this repo, I want to know how to build edge, and the link provided here buries this info.

File: default.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <manifest>
     <!-- defaults for entries that haven't specified all required fields-->
-    <default revision="1.0.0" />
+    <default revision="dev" />
```
Comment (costanic) [inline default.xml:4]: default should be master, correct?

File: README.md (hunk 1 above)
Comment (ygoyal18) [reply README.md:5]: That link has not been updated yet. The new docs are at staging and waiting for 2.0 to be merged to master. This doc have been migrated to [internal docs repos](https://github.com/ARMmbed/pelion-dm-edge-docs/blob/master/docs/quick-start.md).

File: default.xml (hunk 2 above)
Comment (ygoyal18) [reply default.xml:4]: Should we update this after it's merged to master? Or should I branch out of dev, add a commit to change this and then merge to master?

**Pull request merged**